pom.xml

```
@@ -23,7 +23,7 @@
   </parent>
 
   <artifactId>google-compute-engine</artifactId>
-  <version>4.3.18-SNAPSHOT</version>
+  <version>${changelist}</version>
```

possibly we may want to keep the leading `4.` as we have done for the other CD enabled google-plugins
```suggestion
  <version>${revision}.${changelist}</version>
```

[e.g. https://github.com/jenkinsci/google-oauth-plugin/blob/develop/pom.xml#L30](https://github.com/jenkinsci/google-oauth-plugin/blob/b39c5dbe3041767e75c74730739901ef13b282fc/pom.xml#L30)